packages/graph-engine/src/nodes/object/pathExtractor copy.ts

```diff
@@ -0,0 +1,65 @@
+import { AnySchema, ObjectSchema, StringSchema } from "../../schemas/index.js";
+import { INodeDefinition, ToInput, ToOutput } from "../../index.js";
+import { Node } from "../../programmatic/node.js";
+
+export default class ObjectPathExtractorNode extends Node {
```

Comment:
We shouldn't need this anymore with https://github.com/tokens-studio/graph-engine/pull/369